### ISSUE TITLE
Tem modo escuro no site do curso em video? 

### ISSUE THREAD

**Issue by JaquelinePaes** [gustavoguanabara/html-css#174]
Não encontrei a opção do modo escuro no site do curso em video. Se não tiver, fica aqui a sugestão, eu estudo todos os dias e sair do VSC para o site doi demais. 

**F**

**Comment by lucasrenandns** [comment]
@JaquelinePaes tambêm não encontrei, acho que não existe esse modo no site.

**Comment by CEOofDolphins** [comment]
Nope, mas tu pode usar a extensão do Dark Reader que transforma todos os sites em modo escuro

**Comment by flameastro** [comment]
@JaquelinePaes Não tem. Feche a issue.
